docs/framework/interop/com-callable-wrapper.md

```diff
@@ -134,9 +136,14 @@ End Class
 ```  
   
 ```csharp
+// The CLR does not expose a class interface for this type.
+// COM clients can call the members of this class using the methods from the IExplicit interface.
 [ClassInterface(ClassInterfaceType.None)]
-public class LoanApp : IExplicit {
-    public void M() {}
+public class LoanApp : IExplicit
+{
+    public LoanApp() {}
+
+    Int32 IExplicit.M() { return 0; }
```

Comment:
I think it's more idiomatic to use `int`, rather than `Int32`. Also, you could use expression-bodied method here:

```suggestion
    int IExplicit.M() => 0;
```

The same applies for the block below.

Reply:
Here I would choose a classic implementation. Because the 'return' instruction is more recognizable and identifiable.

Reply:
@sharpist I think the 'classic implementation' argument is valid here, but the type really should be `int`.